src/main/java/uk/gov/pay/products/resources/PaymentResource.java

```diff
@@ -32,8 +32,8 @@
     public static final String PAYMENTS_RESOURCE_GET_PAYMENTS = API_VERSION_PATH + "/api/products/{productId}/payments";
 
 
-    private final PaymentsFactory paymentsFactory;
-    private final ProductsFactory productsFactory;
+    private final PaymentFactory paymentsFactory;
```

Comment:
This filed should be `paymentFactory`, mind the `s`

Reply:
Yes, I will blame IntelliJ for not doing the refactoring as one would expect :-) I have changed that.

Reply:
👍